portal/config/eproms/Organization.json

```diff
@@ -2164,7 +2169,7 @@
         }
       ],
       "id": 28000,
-      "language": "pt_BR",
+      "language": "en_US",
```

Comment:
I think this should probably be be added as an option (currently just the default for `id=28000`), by adding it to the `extension` array.